src/services/fileStorage/proxy-service.js

```diff
@@ -1080,14 +1060,11 @@ module.exports = function proxyService() {
 	app.use('/fileStorage/bucket', bucketService);
 	app.use('/fileStorage/total', fileTotalSizeService);
 	app.use('/fileStorage/copy', copyService);
```

Comment:
Einen Aufruf von diesem Endpunkt hab ich im Client nicht gefunden. Kann der auch entfernt werden?

Reply:
Das ist ja ein ziemlich allgemeiner Endpunkt. Deswegen bin ich mir das unsicher, ob wir den nicht erstmal behalten, auch wenn wir ihn gerade nicht nutzen. Aber in Kopieren von Files im Legacy Files Service ist mir auch nicht bekannt. Und "verschieben" scheint auch nicht zu gehen.
![1](https://user-images.githubusercontent.com/106820326/220318516-e8b33d70-1ce0-4500-91c5-0aa24561eec8.jpg)

Reply:
@SevenWaysDP Was denkst du?

Reply:
Legacy file storage ist noch aktive, vllt. das soll nicht Bestandteil unsere Aufräumarbeit sein.